### PR TITLE
fix: MD033/no-inline-html

### PR DESCRIPTION
**Pull request by nschonni**
- Replace <br> with actual line break
- List<T> to System.Collections.Generic.List<T>

### CODE REVIEW

File: docset/winserver2012r2-ps/hpc/Get-HpcTestDetail.md [hunk 1]
```diff
@@ -146,8 +146,16 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## OUTPUTS
 
-### HpcTestCaseDetail, HpcTestCase<Type>Parameter, HpcVariable, HpcTestCaseStep
-This cmdlet returns an **HpcTestCaseDetail** object, zero or more **HpcTestCase\<Type\>Parameter** objects, zero or more **HpcVariable** objects, and one to three **HpcTestCaseStep** objects.
+### HpcTestCaseDetail
+
+### HpcTestCase
```
Comment (nschonni) [inline docset/winserver2012r2-ps/hpc/Get-HpcTestDetail.md]: Not sure about this one. `HpcTestCase<Type>Parameter` appears incorrect, but wasn't sure if it should be a list type

Comment (kenwith) [reply docset/winserver2012r2-ps/hpc/Get-HpcTestDetail.md]: hmm, not sure too, I will have to track down a product team dev to find out

**Comment by kenwith** [comment]
Looks like some conflicts. Let me know if I can help.

**Comment by nschonni** [comment]
Rebased and fixed a few find/replace mistakes

File: docset/winserver2012r2-ps/hpc/Get-HpcTestDetail.md (hunk 1 above)
Comment (kenwith) [reply docset/winserver2012r2-ps/hpc/Get-HpcTestDetail.md]: I have a project to use the cmdlet updater tool I use for Office PowerShell and will use it over here to audit these..  here is tool just FYI.. it is still being iterated on but useful already.. it uses PlatyPS open source tool but streamlines submitting PR's. https://github.com/MicrosoftDocs/office-docs-powershell/tree/master/tools/office-cmdlet-updater

**Pull request merged**